index.bs

```diff
@@ -880,9 +881,9 @@ The getter of the {{Sensor/timestamp!!attribute}} attribute returns
     The {{Sensor/start()}} method must run these steps:
 
     1.  Let |sensor_state| be the value of |sensor_instance|.{{[[state]]}}.
-    1.  If |sensor_state| is either "activating"
-        or "activated", then return.
-    1.  Set |sensor_instance|.{{[[state]]}} to "activating".
+    1.  If |sensor_state| is either "activating" or "activated",
+        then return.
+    1.  [=Queue a task=] to set |sensor_instance|.{{[[state]]}} to "activating".
```

Comment:
The state is not yet 'activating', thus, consecutive start() invocations will queue multiple state change tasks + invoke register sensor multiple times. @pozdnyakov?